From 827bd94d04e46fd84770cbdc8cd15795af307e71 Mon Sep 17 00:00:00 2001
From: "Kevin F. Haggerty" <haggertk@lineageos.org>
Date: Fri, 16 Nov 2018 13:34:33 -0700
Subject: [PATCH 5/5] common: Expand labeling of sysfs_vibrator nodes using
 regex

* In particular, many Samsung kernels use pwm_*-based nodes for
  manipulation. As opposed to chasing all the variations, we should
  just assume that anything under the vibrator directory is a
  vibrator node.

Change-Id: I64824bb59c2103362e6875947840faf378b0dc48
---
 common/private/file_contexts | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/common/private/file_contexts b/common/private/file_contexts
index d48fa49..82229a5 100644
--- a/common/private/file_contexts
+++ b/common/private/file_contexts
@@ -27,7 +27,5 @@
 /system/etc/init\.d/90userinit          u:object_r:userinit_exec:s0
 
 # Vibrator control
-/sys/devices/virtual/timed_output/vibrator/vtg_level u:object_r:sysfs_vibrator:s0
-/sys/devices/virtual/timed_output/vibrator/vtg_min u:object_r:sysfs_vibrator:s0
-/sys/devices/virtual/timed_output/vibrator/vtg_max u:object_r:sysfs_vibrator:s0
+/sys/devices/virtual/timed_output/vibrator(/.*)? u:object_r:sysfs_vibrator:s0
 
-- 
2.17.1

